docs/tracker/tracker_bytetrack.py

```diff
@@ -0,0 +1,102 @@
+
+from typing import List
+
+import numpy as np
+from onemetric.cv.utils.iou import box_iou_batch
+from tqdm.notebook import tqdm
+from ultralytics import YOLO
+
+from supervision import Detections
+from supervision.detection.annotate import BoxAnnotator
+from supervision.tracker.byte_tracker.byte_tracker import byte_tracker, strack
+from supervision.utils.video import VideoInfo, VideoSink, get_video_frames_generator
+
+
+# converts Detections into format that can be consumed by match_detections_with_tracks function
+def detections2boxes(detections: Detections) -> np.ndarray:
+    return np.hstack((
+        detections.xyxy,
+        detections.confidence[:, np.newaxis]
+    ))
+
+# converts List[strack] into format that can be consumed by match_detections_with_tracks function
+def tracks2boxes(tracks: List[strack]) -> np.ndarray:
+    return np.array([
+        track.tlbr
+        for track
+        in tracks
+    ], dtype=float)
+
+# matches our bounding boxes with predictions
+def match_detections_with_tracks(
+    detections: Detections,
+    tracks: List[strack]
+) -> Detections:
+    if not np.any(detections.xyxy) or len(tracks) == 0:
+        return np.empty((0,))
+
+    tracks_boxes = tracks2boxes(tracks=tracks)
+    iou = box_iou_batch(tracks_boxes, detections.xyxy)
+    track2detection = np.argmax(iou, axis=1)
+
+    tracker_ids = [None] * len(detections)
+
+    for tracker_index, detection_index in enumerate(track2detection):
+        if iou[tracker_index, detection_index] != 0:
+            tracker_ids[detection_index] = tracks[tracker_index].track_id
+
+    return tracker_ids
+
+model = YOLO('yolov5s.pt')
```

Comment:
The file yolov5s.pt is directly used in the script. We might want to check if this file exists before running.

Reply:
I don't want us to have that script inside the project. We should aim for a similar format of documentation as with all other `supervision` features. Take a look here: https://roboflow.github.io/supervision/dataset/core/

Reply:
It is ultralytics repo and it downloads automatically the pre-trained model. I added that one as a sample. We can remove the test files and add to another doc if you want.

Reply:
Hi @dbroboflow 👋🏻! We don't store example scripts in supervision. But we document API, just like in the link I provided. If you look there, we also have examples in the docs. We just format those in a different way.

Reply:
Hi @SkalskiP! I updated the tracker as update_from_numpy and update_from_detections. When we are merging it we can delete the samples. I don't delete these files, when you tested it again. I will remove it.